### ISSUE TITLE
Limit annotations to specific graphs and show metrics for all graphs

### ISSUE THREAD

**Issue by mdiorio** [grafana/grafana-zabbix#184]
Love the zabbix plugin for Grafana.  It's absolutely amazing.  Thank you.

I'd love it if we can restrict certain annotations to specific graphs.  If I have a dashboard for a host with CPU, Memory and Network graphs, with annotations for CPU and Memory, I'd like to only see the CPU triggers on the CPU graph, Memory triggers on Memory graph, etc.

I can see that there may occasionally be a benefit to seeing all triggers on all graphs simultaneously when researching an issue historically.

Also, would it be possible to provide an option when using linked cross hairs to show metric data for points on each graph simultaneously?  

Thanks!

**Comment by alexanderzobnin** [comment]
Hi! Thanks for feedback!
Unfortunately, this is Grafana feature only. And now you can't bind annotation to specified panel. But I also want to use this feature, and I'll create relative issue in grafana.

**Comment by tatianab85** [comment]
Hi, was it this feature added? 

Thanks.

**Comment by alexanderzobnin** [comment]
Yes, it was implemented in grafana, so I can try to add this to Zabbix plugin.

**Comment by gustavoghpf** [comment]
Hi, do you still intend to implement this feature?
It would be a great function

Thanks for the amazing plugin!
